src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -64,15 +73,71 @@ void Tab::SetFocused(bool focused)
     }
 }
 
-GUID Tab::GetProfile() const noexcept
+// Method Description:
+// - Returns nullopt if no children of this tab were the last control to be
+//   focused, or the GUID of the profile of the last control to be focused (if
+//   there was one).
+// Return Value:
+// - nullopt if no children of this tab were the last control to be
+//   focused, else the GUID of the profile of the last control to be focused
+std::optional<GUID> Tab::GetFocusedProfile() const noexcept
 {
-    return _profile;
+    return _rootPane->GetFocusedProfile();
 }
 
+// Method Description:
+// - Attempts to update the settings of this tab's tree of panes.
+// Arguments:
+// - settings: The new TerminalSettings to apply to any matching controls
+// - profile: The GUID of the profile these settings should apply to.
+void Tab::UpdateSettings(const TerminalSettings& settings, const GUID& profile)
+{
+    _rootPane->UpdateSettings(settings, profile);
+}
+
+// Method Description:
+// - Focus the last focused control in our tree of panes.

```

Comment:
Here's another part where the comment inconsistency shows. Some in this file have the whole thing, some have an abbreviated doc.